README.md

```diff
@@ -61,6 +61,11 @@ map ((+1) . (*2)) [0..9] -- alternative thanks to Alexandru Dinu
 ```ijs
 1+2*i.10
 ```
+**C++**:
+```c++
+for(int i=0;i<10;i++){
+cout<<(2 * i )+ 1<<"\n";}
```

Comment:
If you want to add C++, can we add a C++20 ranges version? With a compiler explorer link?

Reply:
like this? 
```c++
#include <ranges>
#include <algorithm>
#include <iostream>
int main()
{
    std::ranges::for_each(std::ranges::iota_view(0,10),[](auto i){std::cout << (2 * i)+1 << '\n';});
}
```
https://godbolt.org/z/orMvrn

Reply:
Close, `views::iota` and `views::transform` are preferable imo.